engine/table/src/main/java/io/deephaven/engine/table/impl/select/ConditionFilter.java

```diff
@@ -771,6 +775,16 @@ public ConditionFilter renameFilter(Map<String, String> renames) {
 
     @Override
     public boolean permitParallelization() {
+        // If we have a vectorizable Python function or any Python inputs, then we must check for free-threaded Python
+        // before using the default value of statelessness.
+        final boolean usesPython =
+                pythonFilter || usedInputs.stream().anyMatch(pp -> PyCallableWrapper.class.isAssignableFrom(pp.second)
+                        || PyObject.class.isAssignableFrom(pp.second));
```

Comment:
I don't know the python classes well, does this test include python primitives (strings, numbers)?